libcontainer/cgroups/systemd/unified_hierarchy.go

```diff
@@ -269,23 +259,20 @@ func joinCgroupsV2(c *configs.Cgroup, pid int) error {
 	return createCgroupsv2Path(path)
 }
 
-func (m *UnifiedManager) Freeze(state configs.FreezerState) error {
-	path, err := getSubsystemPath(m.Cgroups, "freezer")
-	if err != nil {
-		return err
-	}
-	prevState := m.Cgroups.Resources.Freezer
-	m.Cgroups.Resources.Freezer = state
-	freezer, err := unifiedSubsystems.Get("freezer")
+func (m *UnifiedManager) fs() (cgroups.Manager, error) {
```

Comment:
Using "fs" here is confusing, given it's also the name of a package... Can you use somthing else instead?

Reply:
changed to `fsManager`